test/integration/lmtp/serverspec/lmtp_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe 'osl-imap::lmtp' do
+  it_behaves_like 'dovecot'
+end
+
+describe file '/etc/dovecot/dovecot.conf' do
+  its(:content) { should match(/^protocols = imap lmtp pop3$/) }
+end
+
+describe file '/etc/dovecot/conf.d/10-master.conf' do
+  its(:content) do
+    should match %r{service lmtp {
+  unix_listener /var/spool/postfix/private/dovecot-lmtp {
+    group = postfix
+    mode = 0600
+    user = postfix
+  }
+}}
+  end
+end
+
+describe file '/var/spool/postfix/private/dovecot-lmtp' do
+  it { should be_socket }
+  it { should be_mode 600 }
+  it { should be_owned_by     'postfix' }
+  it { should be_grouped_into 'postfix' }
+end
+
+describe file '/etc/postfix/main.cf' do
+  its(:content) { should match %r{^local_transport = lmtp:unix:private/dovecot-lmtp$} }
+  its(:content) { should match %r{^virtual_transport = lmtp:unix:private/dovecot-lmtp$} }
+end
+
+describe command 'postconf local_transport virtual_transport' do
+  its(:stdout) { should match %r{^local_transport = lmtp:unix:private/dovecot-lmtp$} }
+  its(:stdout) { should match %r{^virtual_transport = lmtp:unix:private/dovecot-lmtp$} }
+end
+
+describe file '/var/log/maillog' do
+  its(:content) { should match(/dovecot: lmtp\(foo\): .* saved mail to INBOX$/) }
+end
```

Comment:
Any reason you shouldn't use the same test you did in default using expect? It seems to work when I tried it manually.

Reply:
I was testing the log because it indicates that the mail was being delivered via LMTP (which is what this suite is testing). I'll add the expect script in addition to this test.